libs/hmpb/render-backend/src/render_ballot.tsx

```diff
@@ -16,20 +16,38 @@ import {
   PPI,
 } from '@votingworks/hmpb-layout';
 import { join } from 'path';
+import { DOMParser, XMLSerializer } from 'xmldom';
 
 const ASSETS_DIR = join(__dirname, '../assets');
 // These font names need to be exactly of the pattern FontName[-Bold] in order
 // for svg-to-pdfkit to find them (see https://github.com/alafr/SVG-to-PDFKit#fonts).
 // We set fontFamily = HelveticaNeue in the SvgPage component in document_svg.tsx.
 const FONTS = ['HelveticaNeue', 'HelveticaNeue-Bold'];
 
+function removeSvgWidthAndHeight(svg: string): string {
+  const parsed = new DOMParser().parseFromString(svg, 'image/svg+xml');
+  parsed.documentElement.removeAttribute('width');
+  parsed.documentElement.removeAttribute('height');
+  return new XMLSerializer().serializeToString(parsed);
+}
+
 // SVG-to-PDFKit doesn't support embedded SVGs, so we hack around it by
 // inlining the SVG contents.
-function InlineSvgImage(props: SvgImageProps): JSX.Element {
-  const imageContents = readFileSync(join(ASSETS_DIR, props.href)).toString(
-    'utf8'
+function InlineSvgImage({
```

Comment:
Mind refreshing my memory on what the relationship is between `SvgImage` in `libs/hmpb/layout/src/document_svg.tsx` and `InlineSvgImage` here in `libs/hmpb/render-backend/src/render_ballot.tsx`, if any?

Reply:
`SvgImage` renders for the browser, `InlineSvgImage` for PDF. It's the one place where the rendering needs to diverge due to peculiarities with the PDF tooling.

Reply:
Though I suppose I could probably converge on the method used by `InlineSvgImage` now that all seals are just text content and remove support for `href`. I will experiment with that in a follow up PR